src/details/MovieDetailsCard.js

```diff
@@ -41,12 +44,12 @@ class MovieDetailsCard extends Component {
                 <td>${this.state.movie.budget.toLocaleString()}</td>
                 <td>${this.state.movie.revenue.toLocaleString()}</td>
               </tr>
-            </tbody> 
-           </table>
+            </tbody>
+           </table>*/}
         </div>
        </div>
       )
-    }   
+    }
   }
 
```

Comment:
I'd like to move the movie details network request from App into the Movie Details Card, ie give MovieDetailsCard its own componentDidMount method. I think that will help us render all that additional information from the server.